cluster-provision/k8s/1.20/manifests/cni.diff

```diff
@@ -46,20 +49,3 @@ index bcc4ee0c..17a045ff 100644
        containers:
          - name: calico-kube-controllers
            image: docker.io/calico/kube-controllers:v3.18.0
-@@ -3869,4 +3881,16 @@ spec:
- ---
- # Source: calico/templates/configure-canal.yaml
- 
-+---
-+apiVersion: crd.projectcalico.org/v1
```

Comment:
Does Calico provide IPv6 by default now?

Reply:
The pool was there only because we need to configure ipv6 natting, but new calico version has a new enviromnet `CALICO_IPV6POOL_NAT_OUTGOING` variable that activates ipv6 nat without creating a pool, also we are still activating IPv6 with `"assign_ipv6": "true"`